src/atoms/Button.jsx

```diff
@@ -0,0 +1,3 @@
+import styled from "styled-components";
```

Comment:
This PR is only for Loader 
File like Button, deleted files and changes to App, Prop types library  should  not be in this PR

Reply:
Fix conflicts with App